src/site/topic13.rst

```diff
@@ -89,8 +85,130 @@ Indexed Bags
 
 * We also will have removes from arbitrary locations
 
+
+Functionality
+=============
+
+* We will keep our functionality of the bag simple
+    * Add stuff
+    * Remove stuff
+    * Check if something is in the bag
+    * Count the number of times something exists in the bag
+    * See if it's empty
+    * Get the size
+    * Get an iterator for the bag
+        * Iterators are handy tools for looping and consistency
+        * More on iterators later
+
+
+.. code-block:: java
+    :linenos:
+
+    import java.util.Iterator;
+
+    public interface Bag<T> {
+
+        void add(T element);
+
+        T remove(T element);
+
+        boolean contains(T target);
+
+        int getCount(T target);
+
+        boolean isEmpty();
+
+        int size();
+
+        Iterator<T> iterator();
+    }
+
+
+Sorted Bag Functionality
+------------------------
+
+* We want our sorted bags to be bags and have all the functionality as the bag
+    * Add stuff
+    * Remove stuff
+    * Check if something is in the bag
+    * Count the number of times something exists in the bag
+    * See if it's empty
+    * Get the size
+    * Get an iterator for the bag
+
+* However, there will be some functionality specific to the sorted bag that we will want
+    * We have to make sure our `add` adds stuff to the proper location in the bag
```

Comment:
Needs to be two backticks for `add` ?

Occurs throughout the diff.

Reply:
Got it. Good catch.